lib/MusicBrainz/Server/Edit.pm

```diff
@@ -332,6 +332,7 @@ sub TO_JSON {
         display_data => $self->display_data,
         data => $self->data,
         edit_kind => $self->edit_kind,
+        edit_notes => [map { $_->TO_JSON } $self->all_edit_notes],
```

Comment:
```suggestion
        edit_notes => to_json_array($self->edit_notes),
```

Reply:
Similar stuff had to happen for the React.embeds, confused me for a bit :) Hopefully it's all working now.